README.md

```diff
@@ -343,6 +344,15 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+### .node-version
+
+For a little compatability with other node version managers, nvm will also sniff for `.node-version` files. They're the same as `.rmvrc`, they just share a common name.
+
+$ echo "5.9" > .node-version
```

Comment:
From the linked issue, i think `.node-version` might need to be a *specific* version - including the patch.

Reply:
I'm not sure what you mean? nodenv doesn't require a patch?

Reply:
Regardless, node-version will be a subset of nvmrc - so it'll be important to be specific.

Reply:
To be clear here, nodenv is also exact-match with the .node-version string. The _only_ munging that it does, is to allow the addition of a `v` prefix.

Patch-less (or even major-only) version specifiers is only supported via the nodenv-aliases plugin. (And that's because the aliases are created by means of symlinks; ergo, as far as nodenv core is concerned, the exact-match name rules still apply.)